src/test/assembly/x86_64-fortanix-unknown-sgx-lvi-inline-assembly.rs

```diff
@@ -26,11 +26,13 @@ pub extern fn get(ptr: *const u64) -> u64 {
 // CHECK-NEXT: .end_inline_asm
 
 #[no_mangle]
-pub extern fn myret() {
+pub extern "C" fn myret() {
     unsafe {
-        asm!(".start_myret_inline_asm:
+        asm!(
+            ".start_myret_inline_asm:
             ret
-            .end_myret_inline_asm:");
+            .end_myret_inline_asm:"
+        );
```

Comment:
If you're going to change the formatting of this anyway, consider changing it to:
```rust
        asm!(
            ".start_myret_inline_asm:",
            "ret",
            ".end_myret_inline_asm:",
        );
```